Makefile

```diff
@@ -87,6 +87,10 @@ docker-build-local:
 helm:
 	$(MAKE) -C modules helm
 
+.PHONY: helm-chart-push
+helm-chart-push:
+	$(MAKE) -C modules helm-chart-push
```

Comment:
Where is this target used? I couldn't find usage

Reply:
Good point. I thought I did but I use the target from modules directly. I'll remove it.